pom.xml

```diff
@@ -36,7 +36,9 @@
 		<!-- SpotBugs is a program which uses static analysis to look for bugs in Java code -->
 		<!-- Plugin maintenance can be improved by increasing the depth of spotbugs analysis. -->
 		<spotbugs.effort>Max</spotbugs.effort>
+		<spotbugs.failOnError>true</spotbugs.failOnError>
 		<spotbugs.threshold>Low</spotbugs.threshold>
+		<spotbugs.excludeFilterFile>${project.basedir}/src/spotbugs/spotbugs-excludes.xml</spotbugs.excludeFilterFile>
```

Comment:
If you'll rename the file to use the name that is listed in the [tutorial](https://www.jenkins.io/doc/developer/tutorial-improve/add-more-spotbugs-checks/#add-a-spotbugs-exclusions-file-if-needed), `src/spotbugs/excludesFilter.xml`  you won't need to add this line.
```suggestion
```